python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -2245,7 +2279,9 @@ private module Django {
         not exists(regex.getGroupName(_, _)) and
         // first group will have group number 1
         result =
-          routeHandler.getArg(routeHandler.getRequestParamIndex() + regex.getGroupNumber(_, _))
+          routeHandler
+              .getArg(routeHandler.getFirstPossibleRoutedParamIndex() - 1 +
+                  regex.getGroupNumber(_, _))
         or
```

Comment:
I might have written the `-1` at the end because I think of it as shifting group numbers from 1-based to 0-based.

Reply:
that would probably have made more sense :laughing: